client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -186,15 +186,14 @@ object CapiModelEnrichment {
         (tagExistsWithId("tone/comment")(content) && isPillar("News")(content)) ||
           (tagExistsWithId("tone/letters")(content) && isPillar("News")(content)) ||
           isPillar("Opinion")(content)
-      val isCulture: ContentFilter = content => isPillar("Arts")(content) || isPillar("Books")(content)
```

Comment:
We think "books" no longer exists, but doesn't CAPI still use "arts" to represent culture?

Reply:
Ah, good catch! I wasn't aware this was a thing. Will correct it